### ISSUE TITLE
Define Argument and Parameter

### ISSUE THREAD

**Issue by myfonj** [hemanth/functional-programming-jargon#69]
If I understand correctly, the therm `argument` is used here both for local variable name set in function declaration (or expression) phase and its value passed into function in execution phase.  It probably doesn't make a significant diference in context this dictionary (or functional programming in general(?)), but some academic nitpickers might be upset about lack of definition at least.

https://www.coati.io/blog/parameter_or_argument/
tl;dr: **P**arameter = **P**laceholder ; **A**rgument = **A**ctual value

**Comment by jethrolarson** [comment]
I think this will only confuse the intended audience.

**Comment by hemanth** [comment]
> I think this will only confuse the intended audience.

I feel the same.
